bin/date

```diff
@@ -22,7 +22,8 @@ my $VERSION = '1.0.4';
 # prefer this if it is defined.
 my $TZ;
 
-run(\@ARGV) unless caller();
+# run if called directly, indirectly, directly par-packed, undirectly par-packed
+run(\@ARGV) if !caller() || caller(0) =~ /PerlPowerTools/ || caller(0) eq 'PAR' || caller(1) eq 'PAR';
```

Comment:
I don't understand why this is needed, but caller(0) is being evaluated twice, 1st by regex then by "eq". Would it be better to cover both PAR and PerlPowerTools in the regex?

Reply:
I like this plan (with some fixes), but let's choose a namespace that only the packer stuff will use just to avoid possible conflicts. Something like PerlPowerTools::Packed that's not already in use.

Reply:
Good ideas

Reply:
> I don't understand why this is needed, but caller(0) is being evaluated twice, 1st by regex then by "eq". Would it be better to cover both PAR and PerlPowerTools in the regex?

There are some layers and iterations of development on this line of code...